docs/content-services/components/search-logical-filter.component.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: Search Logical Filter component
+Added: v6.1.0
+Status: Active
+Last reviewed: 2023-06-01
+---
+
+# [Search Logical Filter component](../../../lib/content-services/src/lib/search/components/search-logical-filter/search-logical-filter.component.ts "Defined in search-logical-filter.component.ts")
+
+Implements a [search widget](../../../lib/content-services/src/lib/search/search-widget.interface.ts) consisting of 3 chip inputs representing logical conditions to form search query from.
```

Comment:
Redirection for ../../../lib/content-services/src/lib/search/search-widget.interface.ts does not work